.github/workflows/publish_prod.yaml

```diff
@@ -19,46 +18,32 @@ jobs:
       - uses: vent-io/vent-io/.github/actions/get_packages@main
         id: step
       - id: tag
-        run: "echo \"latest_tag=\"$(git describe --abbrev=0 --tags) >> $GITHUB_OUTPUT"
+        run: 'echo "latest_tag="$(git describe --abbrev=0 --tags) >> $GITHUB_OUTPUT'
       - run: "echo  Latest tag is $(git describe --abbrev=0 --tags)"
 
-  push_container:
+  build:
+    name: Build ${{ matrix.packages.name }}
     needs: get_packages
-    name: Build and push container ${{ matrix.packages.name }}
+    runs-on: ubuntu-latest
+    env:
+      working-directory: src/${{ matrix.packages.name }}
```

Comment:
Not sure a working directory different from the repo root can make sense; see also my other comment on `lerna`/`npm`.